ecs-cli/modules/cli/compose/project/project.go

```diff
@@ -51,6 +53,9 @@ type Project interface {
 type ecsProject struct {
 	project.Project
 
+	// TODO: use this instead of project.Project
```

Comment:
Nit: This comment shouldn't be here, right?

Reply:
hmm the intention is still to remove project.Project, though you're right the comment is inaccurate. How about we change to "TODO: use this instead of project.Project.ServiceConfigs" ?

Reply:
Hm, currently, the implementation of the `ServiceConfigs()` method happens to call `project.Project.ServiceConfigs`, which, IIRC I think we'll eventually still need to use when parsing v1/v2 since we'll still be instantiating a libcompose.Project for v1/v2 files. I had thought that we were redefining the interface itself, i.e. having `ContainerConfigs` in rather than `ServiceConfigs` as the actual method in the Project interface, since whatever we do with libcompose.Project.ServiceConfigs will be stuffed into `ecsProject.containerConfigs`.

Reply:
Sorry, that was a long-winded way of explaining something you already clarified in your other comment (and know much more about anyway!). Since the original confusion was around the code comment, I'm fine with this PR as is :) 👍